.circleci/config.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           name: run tests
           command: |
             . venv/bin/activate
-            pytest tests
+            python -m pytest --cov=service tests/ 
```

Comment:
when does this fail? any time it goes down?

Reply:
I wonder if it automatically fails if test coverage goes down, or if you need to set a threshold explicitly with e.g. 
--cov-fail-under=MIN

Reply:
thanks! 
its now at e546bf1